v7/jsonfeed/jsonfeed.py

```diff
@@ -102,7 +99,8 @@ def gen_tasks(self):
                                           timeline, feed_url, output_name)
 
             for classification_name, path_handler in self.supported_taxonomies.items():
-                taxonomy = self.site.taxonomy_plugins[classification_name]
+                taxonomy = self.site.taxonomy_plugins.get(classification_name)
+                if not taxonomy: continue
```

Comment:
How was this failing? (Also, please write this in two lines.)

Reply:
It's failing like this:
```
  File "mysite/plugins/jsonfeed/jsonfeed.py", line 102, in gen_tasks
    taxonomy = self.site.taxonomy_plugins[classification_name]
KeyError: 'section_index'
````

I may be in over my head on trying to make quick little changes to get this working. If so it's probably worth just killing this PR.